src/common/config.ts

```diff
@@ -1,6 +1,12 @@
 
-function getEnv() {
-  return process.env['NODE_ENV'];
+function getEnvironmentVariables() {
+  const env:string | undefined = process.env['NODE_ENV'];
```

Comment:
If not set can we default to 'local' or throw an error? It should be set (with a default) or the service shouldn't start (error).

Reply:
This is a good idea. Something like
`const env = process.env['NODE_ENV'] || 'local';`